docs/config/README.md

```diff
@@ -495,6 +496,54 @@ The `buf` module shows the currently installed version of [Buf](https://buf.buil
 symbol = "🦬 "
 ```
 
+## C
+
+The `c` module shows some information about your C compiler. By default
+the module will be shown if the current directory contains a `.c` or `.h`
+file.
+
+### Options
+
+| Option              | Default                                                                     | Description                                                               |
+| ------------------- | --------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| `format`            | `"via [$symbol $version(-$name )]($style)"`                                 | The format string for the module.                                         |
```

Comment:
I just noticed the symbol is supposed to contain the space in all modules, not the main`format`. If you fix all the `symbol`/`format` occurrences, this PR looks good to me.